src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateAllele.java

```diff
@@ -40,8 +41,8 @@
      * All allele list typed as a {@code List<Allele>} as required by some elements in the VCF framework.
      */
     @SuppressWarnings("unchecked")
-    public static final List<Allele> PLAIN_ALL_ALLELES =
-            (List<Allele>) (List) ALL_ALLELES;
+    public static final List<Allele> PLAIN_ALL_ALLELES = ALL_ALLELES.stream().map(a -> (Allele) a).collect(Collectors.toList());
```

Comment:
note i fixed this in #574 as well. threw a Collections.unmodifiableList around it for good measure, but i guess it isn't necessary.